### ISSUE TITLE
abbr HTML tag not whitelisted

### ISSUE THREAD

**Issue by christopher-b** [instructure/canvas-lms#903]
### Summary:

The &lt;abbr&gt; HTML tag, used to enhance content accessibility, should be whiltelisted in the Rich Content Editor, and not stripped out when saving content.
### Steps to reproduce:
1. Create a new "page" in a course. 
2. Open HTML Editor and enter an &lt;abbr&gt; tag. 
3. Save the content. 
### Expected behavior:

The tag should remain in the saved content.

**Comment by simonista** [comment]
Hey @christopher-b, I've got a commit up to fix this, should hit master this week.

**Comment by christopher-b** [comment]
😁  👍  🙇

**Comment by simonista** [comment]
Thanks for the reminder, it's here: 5719c1d45560f25158029bbb9ae21238bd1354e5
